docs: Improve Linux installation documentation

atxtechbro opened the pull request:
This PR enhances the Linux installation documentation by:

- Adding desktop environment integration instructions for AppImage users
- Providing a complete example for creating desktop shortcuts
- Improving the overall user experience for Linux installations

These changes help Linux users better integrate Amazon Q into their workflow and desktop environments.

I acknowledge that by submitting this pull request, I am contributing code to an AWS open source project and agree to the terms of the project's licensing.

🤖 Assisted by [Amazon Q Developer](https://aws.amazon.com/q/developer)

brandonskiser commented on docs/installation/linux.md:
````diff
@@ -2,15 +2,31 @@
+```bash
+curl --proto '=https' --tlsv1.2 -sSf "https://desktop-release.codewhisperer.us-east-1.amazonaws.com/latest/q-x86_64-linux.zip" -o "q.zip"
+unzip q.zip
+sudo mv q /usr/local/bin/
````
Run the install script instead of only moving `q`. We want to decouple the installation for the `qterm` and `q` binaries in the future, but for now let's stick with what we publicly document

brandonskiser commented on docs/installation/linux.md:
````diff
@@ -2,15 +2,31 @@
+```bash
+q login
+```
 
````
Don't delete the "Most developers" line and still include it as a block quote since it's true that most developers just checking out our app (who have no idea what Builder ID or IAM Identity Center even is) would just authenticate with Builder ID

brandonskiser commented:
Thanks for the contribution! Just a couple changes

atxtechbro commented:
@brandonskiser Hi Brandon! Just checking if you've had a chance to review my latest changes addressing your feedback. I've updated the PR with commit 9fad60e based on your suggestions. Let me know if there's anything else needed to get this merged. Thanks!

brandonskiser commented:
Apologies for the late response. Thank you for the contribution!
